bundle/manifests/sailoperator.clusterserviceversion.yaml

```diff
@@ -663,7 +663,7 @@ spec:
                     - ALL
               - args:
                 - --health-probe-bind-address=:8081
-                - --metrics-bind-address=127.0.0.1:8080
+                - --metrics-bind-address=:8080
```

Comment:
hmm, doesn't this mean that it will now be bound to 0.0.0.0:8080 and is thus less safe than before?

Reply:
I have a question. If we are only listening on `127.0.0.1` how will prometheus (or any other client) scrape the metrics? Its a loopback IP, so other clients will not have access to it right?

Reply:
I believe it's because of the rbac proxy sidecar. Metrics should be retrieved via the proxy, and the proxy itself connects to the operator via loopback.

Reply:
More info: https://github.com/brancz/kube-rbac-proxy?tab=readme-ov-file#motivation

Reply:
thanks @luksa, let me take a look.

Reply:
@luksa I understood how we are using kube-rbac-proxy for supporting metrics. I updated the PR and also verified it on dualStack cluster as you can see from the commit message. PTAL.